### DISCUSSION TITLE
Magic areas compatible with Alarmo?

### DISCUSSION THREAD

**Discussion by Bodge-IT** [jseidl/magic-areas#166] [Q&A] [answered]
Seems to me that there is quite a lot of potential symbiosis with [Alarmo](https://github.com/nielsfaber/alarmo) and MA or am I naiive?
I haven't installed Alarmo yet but it's very compelling, should the two work OK together?

**Comment by jseidl** [comment, marked as answer]
That should work pretty well. You can use the area aggregates to trigger the alarm and use the area (global) to arm/disarm. Will take for a spin def.

I currently do that with the generic alarm and some automations but having an integration do that and trash several lines of yaml is always good. Thanks for the tip!

I need to add some of those cool examples to the Wiki
